runbot/models/docker.py

```diff
@@ -177,20 +199,18 @@ def _compute_use_count(self):
         for record in self:
             record.use_count = len(record.bundle_ids) + len(record.referencing_dockerlayer_ids) + len(record.project_ids) + len(record.version_ids)
 
-    @api.depends('template_id.arch_base', 'layer_ids.rendered', 'layer_ids.sequence')
+    @api.depends('layer_ids.rendered', 'layer_ids.sequence')
     def _compute_dockerfile(self):
         for rec in self:
             content = ''
-            if rec.template_id:
-                try:
-                    res = rec.template_id._render_template(rec.template_id.id) if rec.template_id else ''
-                    dockerfile = re.sub(r'^\s*$', '', res, flags=re.M).strip()
-                    create_user = f"""\nRUN groupadd -g {USERGID} {USERNAME} && useradd --create-home -u {USERUID} -g {USERNAME} -G audio,video {USERNAME}\n"""
-                    content = dockerfile + create_user
-                except QWebException:
-                    content = ''
-            else:
-                content = rec.layer_ids.render_layers()
+            layers = rec.layer_ids
+            if rec.parent_id:
+                layers = self.env['runbot.docker_layer'].new({
+                    'name': 'TEMP LAYER',
+                    'layer_type': 'reference_file',
+                    'reference_dockerfile_id': rec.parent_id.id
+                }) + layers
```

Comment:
Maybe that this could be enough, to test

```suggestion
            if rec.parent_id:
                layers = rec.parent_id.layers + layers
```